package.json

```diff
@@ -27,8 +27,11 @@
     "express": "^4.18.1",
     "jsonwebtoken": "^8.5.1",
     "mongodb-memory-server": "^8.9.5",
-    "mongoose": "^6.7.1",
-    "nodemon": "^2.0.20"
+    "mongoose": "^6.6.5",
```

Comment:
why downgrade the version ?

Reply:
No idea I didn't touch that. Probably because this branch is fairly old